src/TaskTupleAwaiter.Tests/TaskTupleAwaiter.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.2;netcoreapp2.1;netcoreapp2.0;netcoreapp1.1;netcoreapp1.0;net452</TargetFrameworks>
```

Comment:
Can you help me understand this? It seems like it should potentially have its own discussion issue and PR.

Reply:
That commit isn't supposed to be in this PR at all - My bad. I'll revert it.

Reply:
Feel free to force push the PR branch if you like to do that.

Reply:
@jnm2 Odd. it's already reverted in my repo. Where did you see this change?

Reply:
I'm not following. Here's master: https://github.com/buvinghausen/TaskTupleAwaiter/blob/master/src/TaskTupleAwaiter.Tests/TaskTupleAwaiter.Tests.csproj#L4

Reply:
Oh, I accidentally commented on a single commit. https://github.com/buvinghausen/TaskTupleAwaiter/pull/16/files/e08940855d1dc6cf6288305db4d47e908a91c0d0#diff-41d6a0a5de8fc708594b672329a0ad32L4

My bad! That explains why I didn't see it earlier.